modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/organization/management/v1/OrganizationManagementSuccessTest.java

```diff
@@ -604,4 +610,278 @@ private void deleteApplication(String applicationId) throws Exception {
 
         oAuth2RestClient.deleteApplication(applicationId);
     }
+
+    @Test(dependsOnMethods = "testDeleteOrganization")
+    public void createOrganizationsForPaginationTests() throws JSONException {
+
+        List<Map<String, String>> organizations = createOrganizations(NUM_OF_ORGANIZATIONS_FOR_PAGINATION_TESTS);
+
+        if (organizations.size() != NUM_OF_ORGANIZATIONS_FOR_PAGINATION_TESTS) {
+            throw new RuntimeException("Failed to create the expected number of organizations for testing pagination.");
+        }
```

Comment:
Do we need this check

Reply:
No, we don't need to. Handled it with an assertion.